__tests__/components/editor/PropertyResourceTemplate.test.js

```diff
@@ -21,7 +21,7 @@ describe('<PropertyResourceTemplate />', () => {
     },
     reduxPath: ['resourceTemplate:test']
   }
-  const testId = jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
+  jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
```

Comment:
Do you want to use this instead, similar to`__tests__/components/editor/ResourceTemplateForm.test.js`?

```
shortid.generate = jest.fn().mockReturnValue('abcd45')
```

Reply:
Ah, yes. Good call. Changed and pushed. Thanks! Resolving.